### DISCUSSION TITLE
Setting skills over 100 with 'setskill' only work once??

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#15333] [Bug reports]
If you set skill to say 200 using console command `setskill all 200`, the first time you repair an item it will almost instant, but the second item you repair will be much slower until you type the command again

I have no idea why

# Steps to reproduce

1. Throw EMP nade in your sub
2. `setskill all 200`
3. Repair one JB (fast)
4. Repair another JB (slow)

**Comment by Regalis11** [comment]
This has been addressed in the latest unstable, and the fix will be included in the next update. :)
